### PR TITLE
Run go get -u and go mod tidy

### PR DESCRIPTION
**Pull request by nikovacevic**
## Release notes headline (Required or include Not Relevant) 
* Updates go dependencies, fixing a CVE.

## How will this PR impact users?
* Fixes a CVE in `golang.org/x/net`

## Does this PR address any GitHub or Zendesk issues?
* Closes https://kubecost.atlassian.net/browse/CORE-131

## How was this PR tested?
* Build image
* Run unit tests
* Manually on-cluster

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,155 +3,169 @@ module github.com/opencost/opencost
...
 require (
-	cloud.google.com/go/bigquery v1.42.0
-	cloud.google.com/go/compute/metadata v0.2.1
-	cloud.google.com/go/storage v1.27.0
+	cloud.google.com/go/bigquery v1.49.0
+	cloud.google.com/go/compute/metadata v0.2.3
+	cloud.google.com/go/storage v1.30.0
 	github.com/Azure/azure-pipeline-go v0.2.3
-	github.com/Azure/azure-sdk-for-go v65.0.0+incompatible
+	github.com/Azure/azure-sdk-for-go v68.0.0+incompatible
```
Comment (nikovacevic) [inline go.mod:10]: Only "major" change is here -- and not really, even.

File: go.mod [hunk 2]
```diff
@@ -3,155 +3,169 @@ module github.com/opencost/opencost
...
+	go.opencensus.io v0.24.0 // indirect
 	go.uber.org/atomic v1.10.0 // indirect
-	golang.org/x/crypto v0.3.0 // indirect
-	golang.org/x/net v0.4.0 // indirect
-	golang.org/x/sys v0.3.0 // indirect
-	golang.org/x/term v0.3.0 // indirect
-	golang.org/x/time v0.1.0 // indirect
+	golang.org/x/crypto v0.7.0 // indirect
+	golang.org/x/mod v0.9.0 // indirect
+	golang.org/x/net v0.8.0 // indirect
```
Comment (nikovacevic) [inline go.mod:152]: This fixes a CVE (which was fixed in `v0.7.0`)

**Review by michaelmdresser — APPROVED** [review]
If it builds, ship it. We should just watch it carefully in nightly in case anything fun crops up in one of our dependencies.

File: go.mod (hunk 1 above)
Comment (michaelmdresser) [reply go.mod:10]: I think (?) that MS doesn't generally follow semver, but I'm not sure if that's also the case for `azure-sdk-for-go`.

**Comment by nikovacevic** [comment]
Yeah, going to close (for now) in favor of https://github.com/opencost/opencost/pull/1783

**Pull request closed without merging**